org.zdoom.GZDoom.yaml

```diff
@@ -0,0 +1,107 @@
+app-id: org.zdoom.GZDoom
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: "3.32"
+command: gzdoom.sh
+
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --share=ipc
+- --socket=pulseaudio
+
+# Where would wads be?
```

Comment:
Please follow what other games do and remove those filesystem access. 
https://github.com/flathub/com.moddb.TotalChaos/blob/master/com.moddb.TotalChaos.yaml#L74

Reply:
I removed the :ro directories. Looking back at our conversation surrounding ~/.config/ they are indeed not required.

Reply:
The other games do download & install the wads required to play the game in `/app/share/games/doom` which is the reason of setting this env variable `--env=DOOMWADDIR=/app/share/games/doom`
The game must work out of the box without asking the user do anything to get it working.

Reply:
GZDoom also requires some wads which are stored in _/app/share/games/doom_.  during compilation, the system creates 4 .pk3 files which have essential compatibility data. As such, the env= parameter is required.

Reply:
Right. I'm not sure how we can handle this as exposing a folder where we would expect the user to install their extra wads files is not the best way of handling this. 
@barthalion What do you suggest doing in such case?

Reply:
It's indeed a bit of a conflict. We package some additional game data, which is a hard dependency, but we must also allow users to add aditional wads.

Right now it is like this: 
- ~/.config/gzdoom => user game data
- /app/share/games/doom/ => application game data